llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3282,6 +3287,8 @@ class VPlan {
   /// Represents the vector trip count.
   VPValue VectorTripCount;
 
+  VPValue VF;
```

Comment:
```suggestion
  /// Represents the vectorization factor of the loop.
  VPValue VF;
```
(belongs to the loop Region rather than VPlan itself)

Reply:
Added thanks!

VF may also be referenced outside the loop region, so probably should be defined at the top-level and used by the region?

Reply:
Agreed, I think ideally the loop region would be a user of both VF, UF (and possibly VF x UF).